DOC: Fix flake8 issues in doc/source/10min.rst

We didn't start validating the format of PEP8 and other code standards in the documentation examples until recently. We still have some files with errors, that we need to skip, and that we should fix, so we can start validating them.

The first step of this issue would be edit `setup.cfg` in the pandas home, and in the `flake8-rst` section, remove from the `exclude` list the file `doc/source/10min.rst`

After that, running the next command will report the errors in the file (note that syntax error usually prevent to validate other errors, and the list of errors to fix can become much longer when the syntax error is fixed (please make sure that you are using `flake8-rst` version `0.7.0` or higher):

```
$ flake8-rst doc/source/10min.rst 
doc/source/10min.rst:68:16: E225 missing whitespace around operator
doc/source/10min.rst:68:16: E999 SyntaxError: invalid syntax
doc/source/10min.rst:68:20: E225 missing whitespace around operator
```

Once all the errors are addressed, please open a pull request with the fixes in the file, and removing the file from `setup.cfg`. If you need to do something that feels wrong to fix an error, please ask in a comment to this issue. Please avoid other unrelated changes, which can be addressed in a separate pull request.

--- a/doc/source/10min.rst
+++ b/doc/source/10min.rst
@@ -51,7 +51,7 @@ Creating a ``DataFrame`` by passing a dict of objects that can be converted to s
                        'F': 'foo'})
    df2
 
-The columns of the resulting ``DataFrame`` have different 
+The columns of the resulting ``DataFrame`` have different
 :ref:`dtypes <basics.dtypes>`.
 
 .. ipython:: python
@@ -65,7 +65,7 @@ will be completed:
 .. ipython::
 
    @verbatim
-   In [1]: df2.<TAB>
+   In [1]: df2.<TAB>  # noqa: E225, E999
    df2.A                  df2.bool
    df2.abs                df2.boxplot
    df2.add                df2.C
@@ -495,7 +495,7 @@ Another example that can be given is:
 Append
 ~~~~~~
 
-Append rows to a dataframe. See the :ref:`Appending <merging.concatenation>` 
+Append rows to a dataframe. See the :ref:`Appending <merging.concatenation>`
 section.
 
 .. ipython:: python
@@ -528,14 +528,14 @@ See the :ref:`Grouping section <groupby>`.
                       'D': np.random.randn(8)})
    df
 
-Grouping and then applying the :meth:`~DataFrame.sum` function to the resulting 
+Grouping and then applying the :meth:`~DataFrame.sum` function to the resulting
 groups.
 
 .. ipython:: python
 
    df.groupby('A').sum()
 
-Grouping by multiple columns forms a hierarchical index, and again we can 
+Grouping by multiple columns forms a hierarchical index, and again we can
 apply the ``sum`` function.
 
 .. ipython:: python
@@ -671,7 +671,7 @@ Convert the raw grades to a categorical data type.
     df["grade"] = df["raw_grade"].astype("category")
     df["grade"]
 
-Rename the categories to more meaningful names (assigning to 
+Rename the categories to more meaningful names (assigning to
 ``Series.cat.categories`` is inplace!).
 
 .. ipython:: python
@@ -720,7 +720,7 @@ See the :ref:`Plotting <visualization>` docs.
    @savefig series_plot_basic.png
    ts.plot()
 
-On a DataFrame, the :meth:`~DataFrame.plot` method is a convenience to plot all 
+On a DataFrame, the :meth:`~DataFrame.plot` method is a convenience to plot all
 of the columns with labels:
 
 .. ipython:: python

--- a/setup.cfg
+++ b/setup.cfg
@@ -76,7 +76,6 @@ exclude =
     doc/source/whatsnew/v0.23.1.rst
     doc/source/whatsnew/v0.23.2.rst
     doc/source/whatsnew/v0.24.0.rst
-    doc/source/10min.rst
     doc/source/basics.rst
     doc/source/contributing_docstring.rst
     doc/source/dsintro.rst

